macs.php

```diff
@@ -0,0 +1 @@
+i<?php include 'data.php';  ?>
```

Comment:
Thx! Fixed. You using it somewhere?

\g
sandb

On Thu, Dec 3, 2015 at 5:08 PM Jeroen De Dauw <notifications@github.com>
wrote:

> In macs.php <https://github.com/sandb/pamela/pull/3#discussion_r46571090>:
>
> > @@ -0,0 +1 @@
> > +i<?php include 'data.php';  ?>
>
> The moment when you realize there is actually a typo here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sandb/pamela/pull/3/files#r46571090>.
>